test.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
```

Comment:
That's a better way than to try this in a Makefile. I really like this approach. Could you please rename the script to `build.sh`, so we can add a `test.sh` to do some tests later with the images.

I think of some CI scripts

* build.sh
* test.sh
* deploy.sh

I will care of the last two scripts ;-)